README.md

```diff
@@ -127,8 +127,12 @@ struct LLMLocalDemoView: View {
                     )
                 )
 
-                for try await token in try await llmSession.generate() {
-                    responseText.append(token)
+                do {
+                    for try await token in try await llmSession.generate() {
+                        responseText.append(token)
+                    }
+                } catch {
+                    // Handle errors ...
```

Comment:
We could add a small tip here to look into `ViewState` as a nice way to handle errors in a simple setup: https://swiftpackageindex.com/stanfordspezi/speziviews/1.6.0/documentation/speziviews#manage-and-communicate-view-state. We should add a documentation to the main [`ViewState`](https://swiftpackageindex.com/stanfordspezi/speziviews/1.6.0/documentation/speziviews/viewstate) type, we only have a larger example for [`OperationState`](https://swiftpackageindex.com/stanfordspezi/speziviews/1.6.0/documentation/speziviews/operationstate)

We could use a [`> [!TIP]`](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts) in the MD files and the similar setup in the [DocC syntax](https://www.swift.org/documentation/docc/other-formatting-options#Add-Notes-and-Other-Asides).

Reply:
@PSchmiedmayer PTAL. I've updated the comments here, as requested above, and have pushed some `ViewState` documentation to https://github.com/StanfordSpezi/SpeziViews/pull/43 for us to iterate on.

Reply:
Thank you; looks great!